doc/manual/source/interaction/web-api.rst

```diff
@@ -117,46 +117,46 @@ Porcelain commands supported
 
 - **Checkout** (-T)
 
-	 **Currently Supported Options:**
+    **Currently Supported Options:**
 
-		a) **branch*** - the name of the branch to checkout
+      a) **branch** - the name of the branch to checkout
```

Comment:
All the options had `*` at the end to indicate that there are special requirements for using them, explained in the note section below. This change has removed the `*` for all of them.